temporal-sdk/src/main/java/io/temporal/common/SearchAttributeKey.java

```diff
@@ -73,6 +74,17 @@ public static SearchAttributeKey<List<String>> forKeywordList(String name) {
         KEYWORD_LIST_REFLECT_TYPE);
   }
 
+  /**
+   * Create a search attribute key for an untyped attribute type.
+   *
+   * <p>This should only be used when the server can return untyped search attributes, for example,
+   * when describing a schedule workflow action.
+   */
+  public static SearchAttributeKey<Payload> forUntyped(String name) {
```

Comment:
We could also make calling `valueSet` and `valueUnset` an error since I am not aware of any valid use case

Reply:
Yeah, that would work, I think we should. We can hide this more in other langs. Arguably we could make the name of this method more unwieldy, but this is probably good.

Reply:
Done